### PR TITLE
New Package: rcmaehl.MSEdgeRedirect version 0.6.4.0

### PR DESCRIPTION
**Pull request by gnpaone**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/54267)

### CODE REVIEW

File: manifests/r/rcmaehl/MSEdgeRedirect/0.6.4.0/rcmaehl.MSEdgeRedirect.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: rcmaehl.MSEdgeRedirect
+PackageVersion: 0.6.4.0
+PackageLocale: en-US
+Publisher: rcmaehl
```
Comment (ItzLevvie) [inline manifests/r/rcmaehl/MSEdgeRedirect/0.6.4.0/rcmaehl.MSEdgeRedirect.locale.en-US.yaml:7]:
```suggestion
Publisher: Robert Maehl Software
```
This must match Control Panel.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/r/rcmaehl/MSEdgeRedirect/0.6.4.0/rcmaehl.MSEdgeRedirect.locale.en-US.yaml (hunk 1 above)
Comment (gnpaone) [reply manifests/r/rcmaehl/MSEdgeRedirect/0.6.4.0/rcmaehl.MSEdgeRedirect.locale.en-US.yaml:7]: Sure 😊, will change that

**Review by ItzLevvie — APPROVED** [review]

**Comment by gnpaone** [comment]
`Azurepipelines` didn't run due to First time contributors thing, anything to do for that?

**Comment by ItzLevvie** [comment]
> `Azurepipelines` didn't run due to First time contributors thing, anything to do for that?

A PSA has been posted: https://github.com/microsoft/winget-pkgs/issues/54601

**Comment by iMonZ** [comment]
Thank you so much! Finally, I can try this tool, and it works wonderful.
Just one thing, it would be great to rename the settings.exe file to something like MSEdgeRedirectSettings.exe because it's confusing when I search for settings and find this.

**Comment by rcmaehl** [comment]
> Thank you so much! Finally, I can try this tool, and it works wonderful. Just one thing, it would be great to rename the settings.exe file to something like MSEdgeRedirectSettings.exe because it's confusing when I search for settings and find this.

Good point, will change

**Pull request merged**